qtop.py

```diff
@@ -789,6 +789,23 @@ def calculate_split_screen_size():
     return term_columns
 
 
+def convert_to_yaml(scheduler, INPUT_FNs, filenames, write_method, commands):
+
+    # for _file, _func in zip(INPUT_FNs, commands):
+    for _file in INPUT_FNs:
+        file_orig, file_out = filenames[_file], filenames[_file + '_out']
+        _func = commands[_file]
+        print 'executing %(_func)s on file %(_file)s' % {'_func': _func, '_file': _file}
```

Comment:
normally, this is a debug statement and should only be printed with debugging on!